docker/spark-cluster-config/michael/request_consumer.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=The ListenBrainz Request Consumer for Spark
+
+[Service]
+WorkingDirectory=/home/request_consumer/listenbrainz-server/
+ExecStart=/home/request_consumer/listenbrainz-server/docker/start-spark-request-consumer.sh
+User=root
```

Comment:
I tried using the request_consumer user but ran into troubles with file permissions errors in /data/tmp directories and HDFS. So leaving it as root for now. Also, request_consumer user doesn't exist on workers nodes so will need to figure out how/if it'll work,

Reply:
Agreed, but this is a good start. Maybe leave a TODO comment here?

Reply:
Done.